
Traceback (most recent call last):
  File "tests/exceptions/source/others/repeated_lines.py", line 22, in <module>
    recursive(10, 10)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 7 more times]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 6 more times]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 5 more times]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 4 more times]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 3 more times]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 2 more times]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 1 more time]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 15, in recursive
    raise ValueError("End of recursion")
ValueError: End of recursion

Traceback (most recent call last):
> File "tests/exceptions/source/others/repeated_lines.py", line 22, in <module>
    recursive(10, 10)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 7 more times]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 6 more times]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 5 more times]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 4 more times]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 3 more times]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 2 more times]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  [Previous line repeated 1 more time]
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
  File "tests/exceptions/source/others/repeated_lines.py", line 15, in recursive
    raise ValueError("End of recursion")
ValueError: End of recursion

[33m[1mTraceback (most recent call last):[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m22[0m, in [35m<module>[0m
    [1mrecursive[0m[1m([0m[34m[1m10[0m[1m,[0m [34m[1m10[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  [Previous line repeated 7 more times]
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m17[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m,[0m [1minner[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  [Previous line repeated 6 more times]
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m17[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m,[0m [1minner[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  [Previous line repeated 5 more times]
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m17[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m,[0m [1minner[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  [Previous line repeated 4 more times]
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m17[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m,[0m [1minner[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  [Previous line repeated 3 more times]
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m17[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m,[0m [1minner[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  [Previous line repeated 2 more times]
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m17[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m,[0m [1minner[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  [Previous line repeated 1 more time]
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m17[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m,[0m [1minner[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m17[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m,[0m [1minner[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m17[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m,[0m [1minner[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m18[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m[1m,[0m [1minner[0m[35m[1m=[0m[1minner[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m17[0m, in [35mrecursive[0m
    [1mrecursive[0m[1m([0m[1mouter[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m,[0m [1minner[0m[35m[1m=[0m[1mouter[0m [35m[1m-[0m [34m[1m1[0m[1m)[0m
  File "[32mtests/exceptions/source/others/[0m[32m[1mrepeated_lines.py[0m", line [33m15[0m, in [35mrecursive[0m
    [35m[1mraise[0m [1mValueError[0m[1m([0m[36m"End of recursion"[0m[1m)[0m
[31m[1mValueError[0m:[1m End of recursion[0m

Traceback (most recent call last):

  File "tests/exceptions/source/others/repeated_lines.py", line 22, in <module>
    recursive(10, 10)
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 10
    │               └ 10
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 9
    │               └ 10
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 8
    │               └ 10
    └ <function recursive at 0xDEADBEEF>
  [Previous line repeated 7 more times]

  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
    │               │                └ 10
    │               └ 10
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 9
    │               └ 9
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 8
    │               └ 9
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 7
    │               └ 9
    └ <function recursive at 0xDEADBEEF>
  [Previous line repeated 6 more times]

  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
    │               │                └ 9
    │               └ 9
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 8
    │               └ 8
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 7
    │               └ 8
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 6
    │               └ 8
    └ <function recursive at 0xDEADBEEF>
  [Previous line repeated 5 more times]

  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
    │               │                └ 8
    │               └ 8
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 7
    │               └ 7
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 6
    │               └ 7
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 5
    │               └ 7
    └ <function recursive at 0xDEADBEEF>
  [Previous line repeated 4 more times]

  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
    │               │                └ 7
    │               └ 7
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 6
    │               └ 6
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 5
    │               └ 6
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 4
    │               └ 6
    └ <function recursive at 0xDEADBEEF>
  [Previous line repeated 3 more times]

  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
    │               │                └ 6
    │               └ 6
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 5
    │               └ 5
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 4
    │               └ 5
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 3
    │               └ 5
    └ <function recursive at 0xDEADBEEF>
  [Previous line repeated 2 more times]

  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
    │               │                └ 5
    │               └ 5
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 4
    │               └ 4
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 3
    │               └ 4
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 2
    │               └ 4
    └ <function recursive at 0xDEADBEEF>
  [Previous line repeated 1 more time]

  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
    │               │                └ 4
    │               └ 4
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 3
    │               └ 3
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 2
    │               └ 3
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 1
    │               └ 3
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
    │               │                └ 3
    │               └ 3
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 2
    │               └ 2
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 1
    │               └ 2
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
    │               │                └ 2
    │               └ 2
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 18, in recursive
    recursive(outer=outer, inner=inner - 1)
    │               │            └ 1
    │               └ 1
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 17, in recursive
    recursive(outer=outer - 1, inner=outer - 1)
    │               │                └ 1
    │               └ 1
    └ <function recursive at 0xDEADBEEF>

  File "tests/exceptions/source/others/repeated_lines.py", line 15, in recursive
    raise ValueError("End of recursion")

ValueError: End of recursion
